clog/clog_test.go

```diff
@@ -209,4 +214,39 @@ func TestLogger(t *testing.T) {
 		err := ToFileAtLevel("!/*&^%$", "debug")
 		So(err, ShouldNotBeNil)
 	})
+
+	Convey("Given a custom handler", t, func() {
+		ctx := context.Background()
+		handler, err := log15.SyslogHandler(syslog.LOG_USER,
+			"wrrunner", log15.LogfmtFormat())
+		So(err, ShouldBeNil)
+
+		Convey("it can log at given level", func() {
+			syslogpath := "/var/log/syslog"
```

Comment:
Did you check to see if there's some kind of golang syslog package that can tell you the log path on the current OS?

Reply:
Yes, I did search for it, couldn't find it so added "if else" branching.